packages/grid/x-data-grid/src/locales/ukUA.ts

```diff
@@ -174,11 +174,11 @@ const ukUAGrid: Partial<GridLocaleText> = {
 
   // Aggregation
   aggregationMenuItemHeader: 'Агрегація',
-  // aggregationFunctionLabelSum: 'sum',
-  // aggregationFunctionLabelAvg: 'avg',
-  // aggregationFunctionLabelMin: 'min',
-  // aggregationFunctionLabelMax: 'max',
-  // aggregationFunctionLabelSize: 'size',
+  aggregationFunctionLabelSum: 'сумма',
+  aggregationFunctionLabelAvg: 'срзнач',
+  aggregationFunctionLabelMin: 'мін',
+  aggregationFunctionLabelMax: 'макс',
+  aggregationFunctionLabelSize: 'розмір',
```

Comment:
```suggestion
  aggregationFunctionLabelAvg: 'сер',
  aggregationFunctionLabelMin: 'мін',
  aggregationFunctionLabelMax: 'макс',
  aggregationFunctionLabelSize: 'кількість',
```
Ref: Google Spreadsheet

<img width="296" src="https://user-images.githubusercontent.com/13808724/237057290-f410e88c-76e4-485e-9496-e7f907ada423.png">